alacritty/src/event.rs

```diff
@@ -1701,6 +1751,14 @@ impl<'a, N: Notify + 'a, T: EventListener> ActionContext<'a, N, T> {
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+pub struct TouchEvent {
+    /// Unique identifier of a finger.
+    pub finger_id: FingerId,
```

Comment:
Calling this a finger isn't necessarily accurate and in my experience they're generally referred to as touch "slots". So `slot_id` would probably be better. If the comment says finger or uses more generic terminology doesn't really matter.